pkg/gatherers/workloads/workloads_info.go

```diff
@@ -175,23 +194,56 @@ func gatherWorkloadInfo(
 		info.PodCount += namespacePods.Count
 	}
 
-	workloadImageResize(info.PodCount)
+	return limitReached, info, imageCh, nil
+}
 
-	records := []record.Record{
-		{
-			Name: "config/workload_info",
-			Item: record.JSONMarshaller{Object: &info},
-		},
+func isPodRunning(pod *corev1.Pod) bool {
```

Comment:
I just realized that this name doesn't make sense. It's actually the opposite right? When a pod is in the `PodSucceeded` or `PodFailed` state then it's definitely not running. Sorry for the confusion. I would call the function `podTerminated` or `isPodTerminated`.

Reply:
`isPodTerminated` seems good.